Sources/Plasma/PubUtilLib/plNetMessage/plNetMsgHelpers.cpp

```diff
@@ -400,29 +400,26 @@ void plNetMsgObjectHelper::WriteVersion(hsStream* s, hsResMgr* mgr)
 ////////////////////////////////////////////////////////
 plNetMsgObjectListHelper::~plNetMsgObjectListHelper()
 {
-    Reset();         
+    Reset();
 }
 
 void plNetMsgObjectListHelper::Reset()
 {
-    int i;
-    for( i=0 ; i<GetNumObjects() ; i++  )
-    {
+    for (size_t i = 0; i < GetNumObjects(); i++) {
         delete GetObject(i);
         fObjects[i] = nullptr;
-    } // for    
+    }
+
     fObjects.clear();
 }
 
 int plNetMsgObjectListHelper::Poke(hsStream* stream, uint32_t peekOptions)
 {
-    int16_t num = GetNumObjects();
-    stream->WriteLE(num);
-    int i;
-    for( i=0 ;i<num  ;i++  )
-    {
+    size_t num = GetNumObjects();
+    stream->WriteLE(int16_t(num));
```

Comment:
Whenever I truncate a `size_t` to 16-bit or smaller, I like to put an `hsAssert` to check that it isn't too large.  Also, probably better to use `uint16_t` rather than `int16_t`, since that's what hsStream actually takes.